regenerate_and_fix_sdk.py

```diff
@@ -0,0 +1,122 @@
+
+# Correct file
+def correct_file(file, verbose):
+    if not file.endswith(".java"):
+        return
+    
+    with open(file) as f:
+        code = f.read()
+    
+    # Fix webkey models
+    for model in WEBKEY_REPLACE_MODELS:
+        code = code.replace("import com.microsoft.azure.keyvault.models.%s;" % model, "import com.microsoft.azure.keyvault.webkey.%s;" % model)
+    
+    # Fix broken javadoc
+    jdocre = re.compile(r'([a-z|A-Z|0-9]*\<.*?\>)( \* \@param)')
+    code = jdocre.sub(r'\2', code)
+    
+    
+    classre = re.compile(r'public class ([a-z|A-Z|0-9]*?)[ \n\r\t]*{')
+    
+    # If this is a model class and not a custom model class, extend our custom code if necessary - additionally add back potentially nuked imports
+    dir, fn = os.path.split(file)
+    if os.path.split(dir)[1] == "models":
+        classes = classre.search(code)
+        if classes is not None:
+            classname = classes.group(1)
+            # check for corresponding custom class
+            if os.path.exists(os.path.join(CUSTOM_MODEL_DIR, classname + ".java")):
+                if verbose:
+                    print("Updating generated '%s' to extend custom '%s'" % (classname, classname))
+                code = classre.sub(r'public class \1 extends com.microsoft.azure.keyvault.models.custom.\1 {', code)
+    
+        # Add back webkey imports which may have been nuked
+        class_usage_re = re.compile(r'[ \t<](' + '|'.join(WEBKEY_REPLACE_MODELS) + ')[ \t>]')
+        needed_classes = list(set(class_usage_re.findall(code)))
+        if len(needed_classes) > 0:
+            if verbose:
+                print("Adding webkey imports for %s to %s" % ( ",".join(needed_classes), fn ))
+            potential_imports = ["import com.microsoft.azure.keyvault.webkey.%s;" % needed for needed in needed_classes]
+            imports = []
+            
+            for i in potential_imports:
+                if i not in code:
+                    imports.append(i)
+                    
+            if len(imports) > 0:
+                # find package statement at beginning of code, splice and add the imports in between
+                package_re = re.compile(r'package (.+?);\n\n')
+                package = package_re.search(code).group(0)
+                before, after = code.split(package, 2)
+                code = before + package + "\n".join(imports) + "\n" + after
+            
+            
+    # Write back to disk
+    with open(file, 'w') as f:
+        f.write(code)
+            
+        
+if run_autorest:
+    print("Running autorest..")
+    if verbose:
+        print(autorest_cmd)
+    subprocess.run(autorest_cmd, check=True, shell=True)
+    move_generated_code(autorest_target_dir, verbose)
+
+if fix_generated_code:
+    print("Fixing generated code..")
+    for path, dirs, files in os.walk(CODE_TARGET_DIRECTORY):
+        for file in files:
+            correct_file(os.path.join(path, file), verbose)
```

Comment:
Perhaps update your filesys walk to ignore the custom directory, as these shouldn't need to be updated at all